dm-setup/student-vm-startup.sh

```diff
@@ -10,6 +10,11 @@ DEPLOYMENT_NAME=`metadata_value "instance/attributes/deployment"`
 apt-get update
 apt-get install -y git kubectl
 
+# Install Docker
+curl -fsSL get.docker.com -o get-docker.sh
+sh get-docker.sh &&
+for user in $(grep '^google-sudoers:' /etc/group | cut -f 4 -d : | awk 'BEGIN { RS= "," } {print}'); do usermod -a -G docker $user;done
```

Comment:
Moving it down still didnt work. Not sure how to fix. One thing I did notice is that there is an && in line 15 that likely shouldnt be there. In the logs I see an error:

```
tup-script: + sh -c apt-get install -y -qq --no-install-recommends docker-ce >/dev/null
tup-script: debconf: unable to initialize frontend: Dialog
tup-script: debconf: (TERM is not set, so the dialog frontend is not usable.)
tup-script: debconf: falling back to frontend: Readline
tup-script: debconf: unable to initialize frontend: Readline
tup-script: debconf: (This frontend requires a controlling tty.)
tup-script: debconf: falling back to frontend: Teletype
tup-script: dpkg-preconfigure: unable to re-open stdin:
 /etc/group: name=docker, GID=999
 /etc/gshadow: name=docker
e=docker, GID=999
tup-script: E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Which means that the install docker command exits non-zero (although installing docker properly) which then makes the next command not run as they are && together.

Reply:
In order to disable prompts we probably need to set:
```shell
export DEBIAN_FRONTEND=noninteractive
```